### PR TITLE
Add instruction in the README to start developping for lemmy-ui

### PR DESCRIPTION
**Pull request by acatton**
It took me 20min to figure all of this, so I think it should be documented to help the next hacker.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,3 +18,12 @@ The following environment variables can be used to configure lemmy-ui:
 | `LEMMY_UI_DEBUG`               | `bool`   | `false`          | Loads the [Eruda](https://github.com/liriliri/eruda) debugging utility.             |
 | `LEMMY_UI_DISABLE_CSP`         | `bool`   | `false`          | Disables CSP security headers                                                       |
 | `LEMMY_UI_CUSTOM_HTML_HEADER`  | `string` |                  | Injects a custom script into `<head>`.                                              |
+
+## Development
+
+1. `git clone https://github.com/LemmyNet/lemmy-ui`
+2. `cd lemmy-ui`
+3. `git submodule update --init`
```
Comment (SleeplessOne1917) [inline README.md:26]: I added a script in package.json, translations:init, to make updating translations easier.

**Review by dessalines — COMMENTED** [review]
Also this should probably be moved to `CONTRIBUTING.md`

File: README.md (hunk 1 above)
Comment (acatton) [reply README.md]: Good point, I changed it to `yarn translations:init`. Thanks.

**Comment by acatton** [comment]
> Also this should probably be moved to `CONTRIBUTING.md`

Makes sense. Done.

**Comment by acatton** [comment]
@alectrocute why the thumbdown?

**Comment by alectrocute** [comment]
> @alectrocute why the thumbdown?
> 
> Thumbdown by alectrocute

Wrong repo for your PR:

https://github.com/LemmyNet/lemmy-docs is where you can find contributor docs, which are published [here](https://join-lemmy.org/docs/en/contributors/01-overview.html).

Contributions to the docs should follow the same format: paragraph form, not a numbered list. It should be accompanied by a short passage explaining the step.

**Comment by acatton** [comment]
All right. Thanks @alectrocute. Sound good :) , I'll do the PR over there then. Cheers!

**Pull request closed without merging**